### PR TITLE
[openwrt] add callbacks to MGMT API call

### PR DESCRIPTION
**Pull request by jinran-google**
This fixes the build error caused by API changes in <https://github.com/openthread/openthread/pull/6887>.

### CODE REVIEW

**Review by bukepo — APPROVED** [review]

**Comment by jinran-google** [comment]
The border router CI failure is being fixed by <https://github.com/openthread/ot-br-posix/pull/966>. Looks like one of the 2 PRs has to be merged without CI passing.

**Comment by jwhui** [comment]
> The border router CI failure is being fixed by #966. Looks like one of the 2 PRs has to be merged without CI passing.

Rebasing on latest main should address the other CI failures.

File: src/openwrt/ubus/otubus.cpp [hunk 1]
```diff
@@ -915,8 +915,8 @@ int UbusServer::UbusMgmtset(struct ubus_context *     aContext,
     {
         otCommissionerStop(mController->GetInstance());
     }
-    SuccessOrExit(
-        error = otDatasetSendMgmtActiveSet(mController->GetInstance(), &dataset, tlvs, static_cast<uint8_t>(length)));
+    SuccessOrExit(error = otDatasetSendMgmtActiveSet(mController->GetInstance(), &dataset, tlvs,
+                                                     static_cast<uint8_t>(length), nullptr, nullptr));
```
Comment (simonlingoogle) [inline src/openwrt/ubus/otubus.cpp:919]:
```suggestion
    SuccessOrExit(error = otDatasetSendMgmtActiveSet(mController->GetInstance(), &dataset, tlvs,
                                                     static_cast<uint8_t>(length), /* aCallback */ nullptr, /* aContext */ nullptr));
```
nit

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Pull request merged**